index.html

```diff
@@ -1,171 +1,184 @@
-          </li>
-          <li>
-            <img src="http://via.placeholder.com/220x100" />
-          </li>
-        </ul>
-      </section>
+        <div id="first-border">
+            <section class="border">
+                <blockquote class="quote">
+                    Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.
+                </blockquote>
+                <p class="from">
+                    Mx. Lorem
+                </p>
+            </section>
+        </div>
 
 
-      <section>
-        <div>
-          <h3>Support</h3>
-          <ul>
-            <li>
-              Demo
-            </li>
-            <li>
-              Support
-            </li>
-            <li>
-              Contact
-            </li>
-            <li>
-              Docs
-            </li>
-          </ul>
-        </div>
-        <div>
-          <h3>About</h3>
-          <ul>
-            <li>
-              Our Mission
-            </li>
-            <li>
-              Careers
-            </li>
-            <li>
-              Get in Touch
-            </li>
-          </ul>
+        <section class="border">
+            <h2 class="heading">Why?</h2>
+            <ul class="why">
+                <li>
+                    <img id="wave" src="http://via.placeholder.com/100x100" />
+                    <strong>Comfortable</strong>
+                    <p>
+                        Et quas molestias excepturi sint occaecati cupiditate non providents, ut aut reiciendis voluptatibus maiores.
+                    </p>
+                </li>
+                <li>
+                    <img src="http://via.placeholder.com/100x100" />
+                    <strong>Robust</strong>
+                    <p>
+                        At vero eos et accusamus et iusto odio dignissimos ducimus qui. Quis autem vel eum iure reprehenderit qui in ea voluptate velit esse quam.
+                    </p>
+                </li>
+                <li>
+                    <img src="http://via.placeholder.com/100x100" />
+                    <strong>Convenient</strong>
+                    <p>
+                        Itaque earum rerum hic tenetur a sapiente delectus.
+                    </p>
+                </li>
+            </ul>
+        </section>
+
+
+        <div class="blockquotes">
+            <section class="border">
+                <blockquote class="quote">
+                    Nemo enim ipsam voluptatem quia voluptas sit aspernatur aut odit aut fugit, sed quia consequuntur magni dolores eos qui ratione voluptatem sequi nesciunt.
+                </blockquote>
+                <p class="from">
+                    Mx. Ipsum
+                </p>
+            </section>
         </div>
-      </section>
+
+        <section>
+            <h2 class="heading">Meet the team</h2>
+            <ul class="team">
+                <div class="first-three">
```

Comment:
as a reader, it's hard for me to interpret where this is coming from in styles.css. perhaps a more descriptive class would be better.